Set Offset Pressure for Altitude calculations

SafwanQ opened the pull request:
-Set to Barometer mode
-Take ambient pressure
-Set internal Offset Pressure
-Switch Back to Altimeter mode

SafwanQ committed:
Set Offset Pressure for Altitude calculations

-Set to Barometer mode
-Take ambient pressure
-Set internal Offset Pressure
-Switch Back to Altimeter mode

ianjfrosst commented on Autopilot/Path Manager/MPL3115A2.c:
```diff
@@ -16,10 +16,29 @@ char initAltimeter() {
+        int lsb = sendMessage(I2C_SLAVE_ADDRESS, ALTITUDE_LSB_REGISTER, 0, 0, READ);
+        
+        long int pressureWhole = (long)msb<<16 | (long)csb<<8 | (long)lsb;
+        pressure whole >>=6;
```
Typo here?

ianjfrosst commented on Autopilot/Path Manager/MPL3115A2.c:
```diff
@@ -16,10 +16,29 @@ char initAltimeter() {
+        
+        float pressure_decimal = (float)lsb/4.0;
+        
+        float pressure = (float)pressure_whole + pressure_decimal;
```
Something is wrong here.

edenfrosst reviewed (dismissed):
Have you tested if this compiles? Looks like your naming for `pressure_whole` is off. Besides that, this is good.

SafwanQ committed:
Typos and Conventions

-pressure_whole variables
-lsb

SafwanQ committed:
Serge wants simpler calibration

-No longer automatically measures ambient pressure
-Calibration based on ambient pressure input by pilots pre-flight
